docs/install/install-knative-gcp.md

```diff
@@ -72,16 +76,19 @@
 
         ```shell
         chmod +x init_control_plane_gke.sh
```

Comment:
Make the file executable in this PR, then remove this line.

Reply:
This file looks like already executable too, as I run the chmod +x locally, nothing in HEAD changed.